### ISSUE TITLE
_thread_keepalive spams to much with timer logs

### ISSUE THREAD

**Issue by dolshevsk** [faust-streaming/mode#46]
## Checklist

- [X] I have included information about relevant versions
- [X] I have verified that the issue persists when using the `master` branch of Mode.

## Steps to reproduce

Hey there, I'm building quite intense streaming application and my logs just spammed with `Timer _thread_keepalive-AIOKafkaConsumerThread woke up too late` with the value between 0.3-0.5 which from my perspective isn't a big problem and doesn't need additional reaction.

## Expected behavior

It would be great if `max_drift` could be configured somehow so it can be set to `1.0` when the lag is inappropriate, or produce a log only when `self.drifting` threshold is reached.

## Actual behavior

Currently there is no option to configure either the `max_drift` or `self.drifting` thresholds.

## Full traceback

```pytb
[2023-09-29 15:41:39,425] [7] [INFO] Timer _thread_keepalive-AIOKafkaConsumerThread woke up too late, with a drift of +0.3595487903803587 runtime=6.0498714447021484e-06 sleeptime=1.3595487903803587
```

# Versions

* Python version: 3.11
* Mode version: 0.3.5
* Operating system: osx

**Comment by asaguaro1** [comment]
I am also having this issue. It consistently produces INFO logs that are notifying me of a < 1s drift with no other consequences that I can tell. Have you attempted any workarounds?

**Comment by wbarnha** [comment]
I agree, it is really a nuisance, let me see what I can do to give us the option to tell the application that drift is inevitable and that we don't care.
